services/web-app/pages/about-carbon/how-carbon-works.mdx

```diff
@@ -67,25 +67,25 @@ as color tokens, type tokens, spacing tokens, the grid, and more.
 **Guidelines** provide direction on critical topics such as accessibility and content and ensure
 consistent user experiences that meet IBM's standards.
 
-**Assets** include re–usable resources such as components, patterns, functions, and templates. Each
-asset has code, designs, and usage guidance.
+**Assets** are reusable resources such as components, patterns, functions, and templates. Assets
+often have usage and accessibililty guidance, as well as styling specifications and code.
 
-**Design kits** are libraries from Sketch and Figma—and other tools such as Adobe XD and Invision.
-They contain components and elements as well as wireframes and guidance to aid designers.
+**Design kits** are the design counterpart to coded assets. Design kits are also used for creating
+wireframes and guidelines.
 
-**Libraries** are 1:1 with code packages. All coded assets belong to a library and have a
-maintainer. Design kits with compatible code also live in libraries.
+**Libraries** package up and distribute assets, as every asset belongs to a library. Libraries have
+compatible design kits.
 
-## Carbon maintainers
+## Maintainers
 
-Carbon core and local systems have `maintainers` that administer governance over content and
-resource visibility. Each resource cites a maintaining team if an IBM team primarily maintains the
-resource. If an individual or informal group is responsible for a resource, it will be listed as
-community maintained.
+Carbon systems have maintainers that administer governance over content and resource visibility.
```

Comment:
I'd ideally add back in  code highlights for `maintainers` and `schema`. The idea is to guide the eye to definitions of new words. I definitely want it for the ecosystem language of `core` and `local`. I can conceded on `maintainer` and `schema`.